### PR TITLE
Make repo README less Express-centric

### PR DESCRIPTION
**Pull request by paulomarg**
### WHY are these changes introduced?

Fixes https://github.com/Shopify/first-party-library-planning/issues/511

Currently, the README for this project is too focused on Express, considering that the session storage packages have no dependency on Express at all.

### WHAT is this pull request doing?

Rearranging the README to be more of a list of packages in the repo than an introduction of the express package.

### CODE REVIEW

**Comment by paulomarg** [comment]
I'm not 100% happy with the visuals in the main README, but I don't know why 🤔

File: README.md [hunk 1]
```diff
@@ -1,9 +1,60 @@
 # `@shopify/shopify-app-js`
 
-This repo is a collection of packages makes it easy for [Express.js](https://expressjs.com/) apps to integrate with Shopify.
+This repo is a collection of packages that make it easy for apps to integrate with Shopify.
+They all build on top of the [@shopify/shopify-api](https://github.com/Shopify/shopify-api-js) package to cover specific use cases.
 
-The main package is `@shopify/shopify-app-express`, which builds on the `@shopify/shopify-api` package and creates a middleware layer that allows the app to communicate with and authenticate requests from Shopify.
+Below are the packages exported from this repo:
```
Comment (mkevinosullivan) [inline README.md:6]:
```suggestion
This mono-repo supports the following packages:
```

**Review by cquemin — APPROVED** [review]
I see what you mean by the visual render of the doc. I think this the limit of what markdown can do and for code snippets its great but for API documentation it is not ideal...

File: packages/shopify-app-express/docs/reference/shopifyApp.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ This function creates an object that contains everything an Express app needs to
 
 `ApiConfigParams` | :exclamation: required when not using the Shopify CLI
 
-All values allowed by the `@shopify/shopify-api` package [when calling `shopifyApi`](https://github.com/Shopify/shopify-api-js/blob/shopify_api_next/README.md#configurations).
+All values allowed by the `@shopify/shopify-api` package [when calling `shopifyApi`](https://github.com/Shopify/shopify-api-js/blob/main/docs/reference/shopifyApi.md).
```
Comment (mkevinosullivan) [inline packages/shopify-app-express/docs/reference/shopifyApp.md:11]: We should add a pattern to the link check to fail if any link that has `blob` or `tree` does NOT have `main` after it.

**Pull request merged**